modules/redis/src/main/scala/zio/redis/api/Connection.scala

```diff
@@ -388,23 +94,8 @@ trait Connection extends RedisEnvironment {
 }
 
 private[redis] object Connection {
-  final val Auth               = "AUTH"
-  final val ClientCaching      = "CLIENT CACHING"
-  final val ClientGetName      = "CLIENT GETNAME"
-  final val ClientGetRedir     = "CLIENT GETREDIR"
-  final val ClientId           = "CLIENT ID"
-  final val ClientInfo         = "CLIENT INFO"
-  final val ClientKill         = "CLIENT KILL"
-  final val ClientList         = "CLIENT LIST"
-  final val ClientPause        = "CLIENT PAUSE"
-  final val ClientSetName      = "CLIENT SETNAME"
-  final val ClientTracking     = "CLIENT TRACKING"
-  final val ClientTrackingInfo = "CLIENT TRACKINGINFO"
-  final val ClientUnblock      = "CLIENT UNBLOCK"
-  final val ClientUnpause      = "CLIENT UNPAUSE"
-  final val Echo               = "ECHO"
-  final val Ping               = "PING"
-  final val Quit               = "QUIT"
-  final val Reset              = "RESET"
-  final val Select             = "SELECT"
+  final val Auth     = "AUTH"
+  final val ClientId = "CLIENT ID"
+  final val Ping     = "PING"
+  final val Select   = "SELECT"
```

Comment:
These are still in use?

Reply:
I forgot to remove `PING`. 

We have `AUTH`, `CLIENT ID`, and `SELECT` still in use.